packages/dcos-ui-update-service/extra/dcos-ui-update-service.socket

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=DC/OS UI Update Service: service for DC/OS Update
```

Comment:
I've been thinking about that. That service doesn't only update the UI, shall we rename it to `DC/OS UI Service`?

Reply:
Sounds good to me. Shall I?

Reply:
Done as a separate commit so that we can easily revert it if we want.